src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -95,7 +95,20 @@ def repro(self, repro_timeout, repro_args):
     logs.log('Running Syzkaller testcase.')
     additional_args = copy.copy(repro_args)
     result = self.run_and_wait(additional_args, timeout=repro_timeout)
-    logs.log('Syzkaller testcase stopped.')
+    reproducible = False
+    if 'all done.' in result.output:
+      regex = re.compile(r'reproduced (\d+) crashes')
+      search = regex.search(result.output)
+      if search and search.group(1):
+        if search.group(1) > '0':
+          reproducible = True
+
+    if reproducible:
```

Comment:
put lines 99-104 into a helper method. so this can look much simpler as:

```python
if self._crash_was_reproducible(result.output):
  logs.log('Repro success')
  ...
else:
 ...

Reply:
+1
Successfully reproduced crash
and
Failed to reproduce crash
slightly better version of message.

Reply:
Done.